Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml

```diff
@@ -28,6 +28,6 @@
     BreachingCharge: 8 # DeltaV - added breaching charges
   # security officers need to follow a diet regimen!
   contrabandInventory:
-    FoodDonutHomer: 12
-    FoodBoxDonut: 2
+#    FoodDonutHomer: 12 # DeltaV - removed for incongruence at maintainer request
+#    FoodBoxDonut: 2 # DeltaV - removed for incongruence at maintainer request
```

Comment:
Sorry but uh

No. Let jokes exist. Please.

Reply:
Unless these specifically going to damage the game by their existence, these should stay.

Reply:
Was told by headmaint that these should get 1984'd. Besides, why is there food in a vending machine that stocks shields, body armour, and flashlights?